packages/docker-build/src/commands/build.ts

```diff
@@ -200,9 +207,11 @@ export default class DockerBuildCommand extends BaseCommand {
             );
           }
 
+          const buildCommand = this.buildKit ? 'buildx build' : 'build';
```

Comment:
This should be an array.

```suggestion
          const buildCommand = this.buildKit ? ['buildx', 'build'] : ['build'];
```